### DISCUSSION TITLE
GSB default slotted content is difficult to center on page

### DISCUSSION THREAD

**Discussion by kiley-mitti** [RocketCommunicationsInc/astro#933] [Feature Requests]
**Describe the issue**
It's not really a bug so much as a nit that we could improve.
When the right side of the default slotted content is a different width than the left side of the GSB content, the content in the default slot is challenging to properly center on the page.

Normally this could be done using flex box but the div around default slot isn't accessible in shadow parts and has a flex-basis of 100%. 

**Current behavior**
It is difficult to center default slotted content on the GSB (such as <rux-clock>) with regard to the entire width of the page. It DOES center in its own element (the div around default slot) just fine.

**Desired behavior**
It would be nice if devs had extra control over the flex box elements that make up the GSB so that it would be easier to do things like center this content.

**Screenshots**
<img width="2065" alt="Screen Shot 2022-11-14 at 3 45 17 PM" src="https://user-images.githubusercontent.com/110477068/201762151-d772a3f5-f67d-48fc-b72a-81172bcfb5e7.png">

**CodeSandbox**
Check out the example on a larger screen: [CodeSandbox](https://codesandbox.io/s/gifted-snowflake-dpv114)

**Environment (please complete the following information):**

- Browser [e.g. chrome, safari] - all
- @astrouxds/astro-web-components version [e.g. 7.0.0] - latest version 

**Additional context**
I think we could just add a couple more shadow parts to accomplish this. We also may want to check out the css on the div around default slot.

**Comment by kiley-mitti** [comment]
We have added two more shadow parts to the global status bar, one surrounding the app meta and the other to the container of the default slot. While this won't automatically center the content of the default slot on the page, it will allow for easier styling so that the default slot can be more easily centered on the page if that is what the developer wishes to do. This change should be released next Thursday 12/01/22

**Reply by micahjones13** [reply]
Wooo! For anyone interested, here's the PR with the changes https://github.com/RocketCommunicationsInc/astro/pull/937.
